### PR TITLE
Added altV Manager

### PR DESCRIPTION
**Pull request by OvileAmriam**
# Please double check your fork for the following:

- [X] Creating this pull request.
- [X] Cover image is uploaded on imgur.
- [X] Cover image ends in `.jpg, .png, or .jpeg`
- [X] Tags of resources only includes tags supplied on front-page

**Fill each step with 'X' if requirements are met.**

### CODE REVIEW

File: resources/OvileAmriam/alt-Manager.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "altV Manager",
+  "url": "https://github.com/OvileAmriam/altV-Manager",
+  "description": "Automated server installer, updater and launcher with Multi resource/categorization support based on MTA.",
+  "tags": ["boilerplate", "javascript"]
```
Comment (Stuyk) [inline resources/OvileAmriam/alt-Manager.json:5]: Missing `"cover"`. Please add one and do not remove the data fields that you don't use.

Also missing `"cli"`

**Review by Stuyk — CHANGES_REQUESTED** [review]

**Review by OvileAmriam — COMMENTED** [review]
Updated

**Review by OvileAmriam — COMMENTED** [review]
Updated

**Review by OvileAmriam — COMMENTED** [review]
Updated

**Review by Stuyk — APPROVED** [review]

**Pull request merged**